client/src/components/categorical/value/index.js

```diff
@@ -408,6 +418,99 @@ class CategoryValue extends React.Component {
     return false;
   }
 
+  renderMiniStackedBar = (categoryValue) => {
```

Comment:
is there a reason this method is created per-instance, rather than per-class?  Ie, why not declare it:

```
renderMiniStackedBar(categoryValue) {
    ... 
}
```